sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1096,6 +1158,8 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+
```

Comment:
Nit: Please avoid unrelated whitespace changes, as it creates a larger patch footprint and more chance of merge conflicts.

Reply:
Done